bin/plugin/restricted/accountInfo

```diff
@@ -270,6 +270,17 @@ if (OVH::Bastion::is_auditor(account => $self)) {
     $ret{'idle_ignore'} = OVH::Bastion::account_config(account => $account, key => OVH::Bastion::OPT_ACCOUNT_IDLE_IGNORE, public => 1) ? 1 : 0;
     osh_info "\n- Account is immune to idle counter-measures: " . ($ret{'idle_ignore'} ? colored('yes', 'green') : colored('no', 'blue'));
 
+    $ret{'max_inactive_days'} = OVH::Bastion::account_config(account => $account, %{OVH::Bastion::OPT_ACCOUNT_MAX_INACTIVE_DAYS()})->value;
+    if (!defined $ret{'max_inactive_days'}) {
+        osh_info "- Maximum number of days of inactivity before account is disabled: (default)";
```

Comment:
Can't you retrieve the default value ?

Reply:
I could, but I didn't want to mix "bastion configuration" (--osh info) with "account configuration" (--osh accountInfo)